### PR TITLE
docs: minor improvements

### PR DESCRIPTION
**Pull request by NicoPennec**
This PR fixes some typos, misspellings and punctuations.

Happy Hacktoberfest 🎃

### CODE REVIEW

File: docs/content/hatchery/openstack.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ $ cds generate  token -g shared.infra -e persistent
 fc300aad48242d19e782a37d361dfa3e55868a629e52d7f6825c7ce65a72bf92
 ```
 
-Edit the CDS [configuration]({{< relref "hosting/configuration.md">}}) or set the dedicated environment variables. To enable the hatchery, just set the API HTTP and GRPC URL, the token freshly generated and the openstack variables.
+Edit the CDS [configuration]({{< relref "hosting/configuration.md">}}) or set the dedicated environment variables. To enable the hatchery, just set the API HTTP and GRPC URL, the token freshly generated and the openStack variables.
```
Comment (bnjjj) [inline docs/content/hatchery/openstack.md]: s/openStack/OpenStack

File: docs/content/hosting/_index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ You need:
 The CDS engine is made of following components:
 
 - API
-- [Hatcheries (local, docker, openstack, swarm, vsphere)]({{< relref "hatchery/_index.md">}})
+- [Hatcheries (local, Docker, OpenStack, Swarm, vSphere)]({{< relref "hatchery/_index.md">}})
```
Comment (bnjjj) [inline docs/content/hosting/_index.md]: add `, Kubernetes`

File: docs/content/hosting/_index.md [hunk 3]
```diff
@@ -38,10 +38,10 @@ Start CDS Engine Services:
  * Hatcheries:
 	They are the components responsible for spawning workers. Supported platforms/orchestrators are:
 	 * Local machine
-	 * Openstack
+	 * OpenStack
 	 * Docker Swarm
-	 * Openstack
-	 * Vsphere
+	 * OpenStack
+	 * vSphere
```
Comment (bnjjj) [inline docs/content/hosting/_index.md]: * Kubernetes

File: docs/content/workflows/design/hooks/git-repo-webhook.md [hunk 4]
```diff
@@ -12,4 +12,4 @@ You have to:
 * link an application to a git repository
 * add a Repository Webhook on the root pipeline, this pipeline have the application linked in the [context]({{< relref "workflows/design/pipeline-context.md" >}})
 
-Github / Bitbucket & Gitlab are supported by CDS.
+GitHub / Bitbucket & GitLab are supported by CDS.
```
Comment (bnjjj) [inline docs/content/workflows/design/hooks/git-repo-webhook.md]: s/GitHub / Bitbucket/GitHub, Bitbucket

File: docs/content/hosting/_index.md (hunk 2 above)
Comment (NicoPennec) [reply docs/content/hosting/_index.md]: done!

**Comment by NicoPennec** [comment]
@bnjjj all your requests are resolved

**Pull request merged**